### DISCUSSION TITLE
theme

### DISCUSSION THREAD

**Discussion by tharindu899** [LazyVim/LazyVim#1527] [Q&A]
https://github.com/NvChad/base46

how to add this theme

**Comment by dpetka2001** [comment]
> This theme plugin is supposed to be used along with [NvChad](https://github.com/NvChad/NvChad) only

Are you sure it can be used outside of NvChad? You should ask on their Discord server to see what they say. Also see this [issue](https://github.com/NvChad/base46/issues/210) on the repo. I guess you should get in contact with the guy that made the plugin to tell you how to use it outside NvChad.
